myawesomecss.css

```diff
@@ -1,70 +1,181 @@
-/* Project Meta */
-
-/* reset  */
-
-/* general */
-
-/* typography */
-
-/* grid */
-
-/* header */
-
-/* footer */
-
-/* Page template A */
-
-/* Page template B */
-
-/* Media Queries */
-
-
-.box {background: #3498db; border: 1px solid black;}
-
-.box {
-    background: #2ecc71;
-    border: 1px solid black;
-    }
-
-
-    #div {
-
-        margin-top: 10px;
-        
-        margin-right: 10px;
-        
-        margin-bottom: 3px;
-        
-        margint-left: 4px;
-        
-        }
-
-
-        
-html,
-
 body {
-
-font-family: 'Lato', sans-serif;
-
-height: 100%;
-
-background: #ecf0f1;
-
-overflow: hidden;
-
+    font-family: 'Century Gothic', Calibri, serif;
+    font-size: 15px;
+    margin: 0px;
+    padding: 0px;
+    display: box;
+    -moz-box-orient: horizontal;
 }
 
-h1 {
 
-color: #3498db;
 
-text-align: center;
+.parent{
+    /* margin-left: 400px 100%; */
+    /* margin-right: 700px 100%; */
+    /* color: rgb(112, 179, 179); */
+    /* border: #000; */
+    margin-left: 70px;
+    margin-right: 70px;
+    /* width: 100%;  */
+    background: rgb(204, 199, 204);
+    
+}
+.enfant1{
+    /* width: 50%; */
+    /* height: 700px; */
+    width: 90%;
+    height: auto;
+    background: blanchedalmond; 
+    float: left;
+    border-bottom: 0px;
+    /* font-size:1.5em; */
+    /* font-weight: 700; */
+    /* margin-left: 5px; */
+    /* margin-right: 5px;  */
+    /* float: left; */
+    /* margin: 10px 3%; */
+    /* margin-left: 10px 5%; */
+    /* margin-right: 10px 5%; */
```

Comment:
[Optional] Your css file has multiple sections of code that are commented out, which is a bad practice. Consider removing all code that is not performing any roles in your code. In general, comments should only be used to document your code and not to disable code that should otherwise be deleted.